test/python/transpiler/test_permrowcol.py

```diff
@@ -194,43 +194,8 @@ def test_choose_column_returns_correct_index_with_similar_col_sums(self):
 
         self.assertEqual(index, 2)
 
-    def test_eliminate_column_returns_list(self):
-        """Test the output type of eliminate_column"""
-        coupling = CouplingMap()
-        permrowcol = PermRowCol(coupling)
-        parity_mat = np.ndarray(0)
-        terminals = np.ndarray(0)
-        circ = QuantumCircuit(0)
-
-        instance = permrowcol._eliminate_column(circ, parity_mat, 0, 0, terminals)
-
-        self.assertIsInstance(instance, list)
-
-    def test_eliminate_column_identity_column(self):
```

Comment:
This and the invalid cnot check for the circuit (instead of the output tuple list) for `_eliminate_row` and `_eliminate_column` would have been good to keep if we need to rewrite/refactor stuff in near future, but we can add them back later if needed